gatling-jms/src/main/scala/io/gatling/jms/check/JmsCheckSupport.scala

```diff
@@ -46,6 +62,28 @@ trait JmsCheckSupport {
   )(implicit materializer: CheckMaterializer[A, JmsCheck, Message, P]): JmsCheck =
     findCheckBuilder.find.exists
 
+  implicit val jmsBodyStringCheckMaterializer: CheckMaterializer[BodyStringCheckType, JmsCheck, Message, String] =
+    JmsBodyStringCheckMaterializer()
+  implicit val jmsBodySubstringCheckMaterializer: CheckMaterializer[SubstringCheckType, JmsCheck, Message, String] =
+    JmsBodySubstringCheckMaterializer()
+
   implicit def jmsXPathmaterializer(implicit xmlParsers: XmlParsers): CheckMaterializer[XPathCheckType, JmsCheck, Message, Option[Dom]] =
     new JmsXPathCheckMaterializer(xmlParsers)
+
+  implicit def jmsJsonPathCheckMaterializer(implicit jsonParsers: JsonParsers): JmsJsonPathCheckMaterializer =
+    new JmsJsonPathCheckMaterializer(jsonParsers)
+
+  implicit val amqpUntypedConditionalCheckWrapper: UntypedConditionalCheckWrapper[JmsCheck] =
```

Comment:
amqp?

Reply:
this is typo, i  implemented  similar functionality in plugin for amqp, and i forget change names of variables

Reply:
this is typo, i  implemented  similar functionality in plugin for amqp, and i forget change names of variables